src/test/java/de/rondiplomatico/nds/NDSTileTest.java

```diff
@@ -298,6 +300,222 @@ public void testCenterWorks() {
         }
     }
 
+    @Test

```

Comment:
What is the test coverage of these tests?

Reply:
NDSTile - 94 %.
nds-tiles repo - 82.9 %.